### PR TITLE
Rewrote eps.py to implement logging messages and error handling

### PR DESCRIPTION
**Pull request by JayChen35**
#18

### CODE REVIEW

**Review by ayusher — DISMISSED** [review]

File: config/config_OLD_default.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+core:
```
Comment (abagali1) [inline config/config_OLD_default.yml]: We dont need to store old copies of this. Everything is already saved in `git`s history

File: config/config_default.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# Simulate = whether or not core is running while running fake data
```
Comment (abagali1) [inline config/config_default.yml:1]: `simulate` is deprecated. This line is unnecessary

File: config/config_default.yml [hunk 3]
```diff
@@ -11,54 +12,12 @@ core:
             - iridium
             - telemetry
     config_save_interval: 60
-    dump_interval: 10
```
Comment (abagali1) [inline config/config_default.yml:14]: keep `dump_interval` under the `core` key b/c it is the interval on which `core` tells `telemetry` its time to send info down in a dump

File: submodules/eps/__init__.py [hunk 4]
```diff
@@ -1,68 +1,140 @@
...
 
 from core.mode import Mode
 from core.threadhandler import ThreadHandler
+from core.error import Error
+from core.log import Log
+from submodules import telemetry
 
 # Initialize global variables
 logger = logging.getLogger("EPS")
 address = 0x57
```
Comment (abagali1) [inline submodules/eps/__init__.py]: Do not initialize variables outside a method scope b/c these variables will be reinitialized whenever the module is imported. (i.e. variables will be reinitialized if two different files import this file). Instead, initialize these variables (and any other global variables) in the `start()` method so they are only initialized once.
```
def start():
    global epsdict
    epsdict = {...}
```

File: submodules/eps/__init__.py [hunk 5]
```diff
@@ -1,68 +1,140 @@
...
 epsdict = {'a': 1, 'i2c': 2, 'c': 3, 'antenna': 4,
            'pi': 5, 'iridium': 6, 'aprs': 7, 'h': 8, 'i': 9, 'j': 10}
-mode = Mode.NORMAL
-
+with open('config/config_default.yml') as stream:
```
Comment (abagali1) [inline submodules/eps/__init__.py]: During the startup sequence, `core` reads from the `config_defaul.yml` file and loads it into memory by storing it inside a global variable in `core` called `config`. Reloading the `config_default.yml` file in each module would be a waste of memory and also removes the functionality of having a `config_custom.yml` file during development. Instead, use `from core import config` and reference the `config` variable when you need to read configuration data

File: submodules/eps/__init__.py [hunk 6]
```diff
@@ -150,26 +240,7 @@ def board_check() -> None:
 
 def start():
```
Comment (abagali1) [inline submodules/eps/__init__.py:232]: Update this method with any global variable initializations listed above.

**Review by abagali1 — CHANGES_REQUESTED** [review]
LOVING the documentation. I made a few changes above, but otherwise great work. I am going to hold off merging these changes until we can run through a full unit test with the eps board. (unless there is a pressing reason to merge these changes now)

**Review by ayusher — DISMISSED** [review]

**Review by abagali1 — DISMISSED** [review]

**Review by abagali1 — APPROVED** [review]

**Pull request merged**